conf.d/main

```diff
@@ -83,14 +83,23 @@ _token=$TOKEN\
 &terms_of_service=on\
 &privacy_policy=on"
 
-echo '  * * * * * www-data php /var/www/invoiceninja/artisan schedule:run >> /dev/null 2>&1' >> /etc/cron.d/invoice-ninja
+echo '  * * * * * root /usr/local/bin/turnkey-artisan schedule:run >> /dev/null 2>&1' >> /etc/cron.d/invoice-ninja
```

Comment:
If this change to using root is intentional, could you add a comment explaining why?

Reply:
Good call. Will do.